app/imports/components/Projects/CreateNewProject/CreateNewProject.jsx

```diff
@@ -53,20 +208,36 @@ class CreateNewProject extends Component {
     }
 
     render() {
-        return (
-            <form onSubmit={this.onSubmit}>
-                <h2>Here you can create a new project!</h2>
-                {this.renderTextField()}
-                {!this.state.isError && this.state.isResult ? '' :
-                <RaisedButton label="Add" type="submit" primary />
-                }
-            </form>
+        return (<div>
+            <h2>Here you can create a new project!</h2>
+            {this.renderProjectNameForm()}
+            {this.renderModForm()}
+            {!this.state.isError && this.state.isResult ? '' :
+            <List>
```

Comment:
What do you think about `Chip` component :) I think that this component is more user friendly for this purpose.